paths/git-trouble/fc-manycommits.md

```diff
@@ -0,0 +1,49 @@
+---
+layout: simple-class
+header:
+  overlay_image: cover.jpeg
+  overlay_filter: rgba(46, 129, 200, 0.6)
+title: Too Many (small) Commits!
+permalink: /git-trouble/03
+next-page: /git-trouble/02
+facilitator: false
+sidebar:
+  nav: "advanced"
+main-content: |
+  While working on your project, you might have made commits for every little change you made along the way and created a commit history that looks more like an avalanche of information as opposed to a succinct list of the changes you made to the branch.
+
+  As with most things, Git comes equipped with the tools necessary tools to save you from yourself.
+
+  Keep in mind, all exercises expect you to have run the script to create files using the scripts found on the [Set Up Your Environment](/on-demand/git-trouble/01) page.
+pushed: |
+    When it comes to gitting (HA!) out of troublesome situations, you might have a handful of tools at your disposal to make everything right. In this particular instance Git has two tools that allow you to modify your commit history. In the example provided, the `reset` command is used, however, the `rebase -i` functionality could also be used. For information on how to use `rebase -i` command, review the "I pushed" section in the [Commit Message Sucks](/on-demand/git-trouble/03) scenario.
```

Comment:
@beardofedu the discussion of `rebase -i` feels weird in this paragraph. After all of the talk about rebase, I expect the steps below to take me through a rebase. Maybe talk about rebase -i after the steps??